lib/elasticsearch_client.py

```diff
@@ -48,7 +48,7 @@ def delete_index(self, index_name, ignore_fail=False):
     def get_indices(self):
         return self.es_connection.indices.stats()['indices'].keys()
 
-    def create_index(self, index_name, ignore_fail=False, index_config=None):
+    def create_index(self, index_name, index_config=None):
```

Comment:
Same with this, this change was merged into master, so I cherry-picked it here since we're technically dependent on it.